docs/aws-creds.md

```
@@ -0,0 +1,90 @@
+With a specific profile:
+
+```yaml
+hooks:
+  - event: PreToolUse
+    type: claude-command-persistent
+    command: "harnx-aws-creds --profile my-profile"
+```
+
+## CLI Reference
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)